exercises/prime-factors/example.jl

```diff
@@ -0,0 +1,34 @@
+function isPrime(n)
+    if n == 1
+        return(false)
```

Comment:
```suggestion
        return false
```

`return` is a keyword, not a function. There are no parantheses around its arguments.